helm-charts/basehub/values.yaml

```diff
@@ -1,3 +1,7 @@
+userServiceAccount:
+  enabled: true
```

Comment:
Should this default to true? I'm not sure if it's better to explicitly enable features or rely on default config